Index: pyglet/clock.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- pyglet/clock.py	(date 1550106713000)
+++ pyglet/clock.py	(date 1574436147000)
@@ -143,6 +143,7 @@
 from builtins import range
 from builtins import object

+import sys
 import time
 import ctypes
 from operator import attrgetter
@@ -163,7 +164,10 @@
         def sleep(self, microseconds):
             time.sleep(microseconds * 1e-6)

-    _default_time_function = time.clock
+    if sys.version_info >= (3, 8):
+        _default_time_function = time.perf_counter
+    else:
+        _default_time_function = time.clock

 else:
     _c = pyglet.lib.load_library('c')
